### DISCUSSION TITLE
Tarea 3 - Informe Parte 1

### DISCUSSION THREAD

**Discussion by lfarr1110** [IIC2133-PUC/2024-2#185] [Tarea 3] [answered]
Buenas, en el enunciado de indica _**"Describe cómo sería una solución utilizando un enfoque "Greedy""**_, mi duda es, ¿hay que describir como sería un enfoque Greedy sin necesariamente realizarlo en la parte del código? ¿o se espera que se resuelva el problema mediante un algoritmo Greedy?. También si en la parte de describir basta con explicarlo con palabras o hay que mostrarlo de forma mas explícita con pseudocódigo (por ejemplo).
De antemano, muchas gracias.

**Comment by sebacarrascop** [comment, marked as answer]
Hola! el código se evalúa a través de tests, se espera lo que esta en el informe se corresponda al código, pero son cosas aparte.
Respecto a la explicación, debe quedar claro, si tú crees que un pseudo código ayudaría a eso, puedes agregarlo, pero no se especifica nada más.
